src/vs/workbench/contrib/terminal/common/scripts/psreadline/cgmanifest.json

```diff
@@ -0,0 +1,38 @@
+{
+  "registrations": [
+    {
+      "component": {
+        "type": "git",
+        "git": {
+          "name": "PSReadLine",
+          "repositoryUrl": "https://github.com/PowerShell/PSReadLine",
+          "commitHash": "abc123"
+        }
+      },
+      "licenseDetail": [
+        "Copyright (c) 2013, Jason Shirk",
```

Comment:
It concerns me that we never updated this to be assigned to Microsoft Corporation 😬